### ISSUE TITLE
Calico on EKS

### ISSUE THREAD

**Issue by jicowan** [projectcalico/typha#151]
Calico daemon fails to start, unable to parse Calico version : is not in dotted-tri format.  
## Current Behavior

```
kubectl logs calico-node-29s2l -n kube-system
2018-04-23 19:21:28.042 [INFO][10] startup.go 248: Early log level set to info
2018-04-23 19:21:28.044 [INFO][10] startup.go 278: Checking datastore connection
2018-04-23 19:21:28.061 [INFO][10] startup.go 302: Datastore connection verified
2018-04-23 19:21:28.061 [INFO][10] startup.go 99: Datastore is ready
2018-04-23 19:21:28.066 [WARNING][10] migrate.go 723: Unable to parse Calico version :  is not in dotted-tri format
2018-04-23 19:21:28.066 [ERROR][10] migrate.go 665: Could not parse CalicoVersion  in ClusterInformation: Error converting version
2018-04-23 19:21:28.066 [ERROR][10] startup.go 106: Unable to ensure datastore is migrated. error=Error converting version
2018-04-23 19:21:28.066 [WARNING][10] startup.go 1007: Terminating
Calico node failed to start
```
## Steps to Reproduce (for bugs)

1. Remove aws-node cni from running cluster
2. Apply https://github.com/aws/amazon-vpc-cni-k8s/blob/master/misc/aws-k8s-cni-calico.yaml

## Your Environment

* Typha version (if using Typha binary directly `calico-typha --version`):
* And/Or, `calico/typha` container version (if running Typha in a container):
* Which orchestrator are you using (e.g. Kubernetes, OpenStack, Docker, Mesos): EKS, Kubernetes
* Orchestrator version (if used): 1.9
* Etcd version (if using etcd datastore driver):
* Operating System and version: Amazon Linux 2
* Link to your project (optional):

**Comment by caseydavenport** [comment]
@jicowan yes, I think you're seeing the same race that was in #118 but has been fixed since.

Could you try updating the manifest you're using to use a newer version of Calico? Looks like it's still on v3.0.2.

v3.0.6 is the latest v3.0 release and includes the fix for #118.
